### PR TITLE
update dev_setup.sh

### PR DESCRIPTION
**Pull request by rithythul**
Add pacman package manager for Arch based environment.

## Motivation

We are an Archlinux people, we want this project to be easily view by other Archian. 
There is no effect to the Libra code base, except allow Arch base system to test more easily.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instrutions for verifying that your changes work.)

### CODE REVIEW

File: scripts/dev_setup.sh [hunk 1]
```diff
@@ -25,6 +25,9 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
 		PACKAGE_MANAGER="yum"
 	elif which apt-get &>/dev/null; then
 		PACKAGE_MANAGER="apt-get"
+	elif which pacman &>/dev/null; then
+		PACKAGE_MANAGER="pacman"
+
 	else
 		echo "Unable to find supported package manager (yum or apt-get). Abort"
```
Comment (frasergr) [inline scripts/dev_setup.sh]: Indicate pacman is one of the supported package managers

**Comment by adsharma** [comment]
Also see #48

**Comment by ericnakagawa** [comment]
Thanks for the PR @rithythul. Our team is reviewing your PR and will reply back soon. We're still sorting out CLAs and will reply back with an update.

**Comment by rithythul** [comment]
@adsharma brought up something better #48

**Pull request closed without merging**